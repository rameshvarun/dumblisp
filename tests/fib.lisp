(defun fib (n)
  (if (or (= n 0) (= n 1))
    1
    (+ (fib (- n 1)) (fib (- n 2)))
    ))

(print "fib(0) =" (fib 0))
(print "fib(1) =" (fib 1))
(print "fib(2) =" (fib 2))
(print "fib(10) =" (fib 10))
