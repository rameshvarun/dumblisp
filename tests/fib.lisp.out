fib(0) = 1
fib(1) = 1
fib(2) = 2
fib(10) = 89
